internal/server/v0/internalplace/golden/get_place_page_data/city.sample.json

```diff
@@ -4554,7 +4554,7 @@
             "2099-12": 3.079742
           },
           "metadata": {
-            "importName": "NASA_NEXDCP30_StatVarSeriesAggr",
```

Comment:
Note: I think we can now remove this here https://github.com/datacommonsorg/mixer/blob/3b0fa2428993be5283afe87cc94643d91df182ce/internal/server/ranking/ranking.go#L68-L69